include/RAJA/pattern/detail/multi_reduce.hpp

```diff
@@ -26,32 +26,29 @@
 #include "RAJA/util/RepeatView.hpp"
 
 
-#define RAJA_DECLARE_MULTI_REDUCER(OP_NAME, OP, POL, DATA)    \
-  template <typename tuning, typename T>                      \
-  struct MultiReduce##OP_NAME<POL<tuning>, T>                 \
-      : reduce::detail::BaseMultiReduce##OP_NAME<             \
-            DATA<T, RAJA::reduce::OP<T>, tuning>>             \
-  {                                                           \
-    using policy = POL<tuning>;                               \
-    using Base = reduce::detail::BaseMultiReduce##OP_NAME<    \
-        DATA<T, RAJA::reduce::OP<T>, tuning>>;                \
-    using Base::Base;                                         \
-    using typename Base::value_type;                          \
-    using typename Base::reference;                           \
-                                                              \
-    RAJA_SUPPRESS_HD_WARN                                     \
-    RAJA_HOST_DEVICE                                          \
-    reference operator[](size_t bin) const                    \
-    {                                                         \
-      return reference(*this, bin);                           \
-    }                                                         \
+#define RAJA_DECLARE_MULTI_REDUCER(OP_NAME, OP, POL, DATA)                     \
+  template<typename tuning, typename T>                                        \
+  struct MultiReduce##OP_NAME<POL<tuning>, T>                                  \
+      : reduce::detail::BaseMultiReduce##OP_NAME<                              \
+            DATA<T, RAJA::reduce::OP<T>, tuning>>                              \
+  {                                                                            \
+    using policy = POL<tuning>;                                                \
+    using Base   = reduce::detail::BaseMultiReduce##OP_NAME<                   \
+        DATA<T, RAJA::reduce::OP<T>, tuning>>;                               \
+    using Base::Base;                                                          \
```

Comment:
I guess something confused its indentation alignment.

Reply:
I think clang-format has problems with making nested templates look pretty and readable, in general. That's why we won't be running it in the examples, exercises, and test directories.